### PR TITLE
Allow hide/reveal on merge controls

### PR DESCRIPTION
**Pull request by jdlrobson**
Fixes: #2248

@hornc note I haven't tested this but hopefully this will address the issue you are seeing!

### CODE REVIEW

**Comment by hornc** [comment]
I'll check this out and test locally, thank you!

**Comment by cdrini** [comment]
Code looks good to me; would you mind assigning yourself to this one @hornc and merging it once you test it?

File: openlibrary/templates/type/author/view.html [hunk 1]
```diff
@@ -91,7 +91,7 @@ <h2 class="author collapse">
 
     $if ctx.user and ("merge-authors" in ctx.features or ctx.user.is_admin()) and query_param('merge', 'false') == 'true':
        <div class="message hidden">
```
Comment (hornc) [inline openlibrary/templates/type/author/view.html]: @jdlrobson I have tested and am not seeing the progress or messages, I think this `hidden` is overriding the visibility of the sub elements? If I remove the `hidden` manually after the page has loaded, I do see just the final status message I would expect to see.

Comment (jdlrobson) [reply openlibrary/templates/type/author/view.html]: Looks like i missed one. This should also be style="display:none"

Is there any easy way for me to get this interface locally. I'd love to be more helpful and test this.

**Comment by jdlrobson** [comment]
Eek i messed up the amendment. Shouldn't have added those extra files... those are for a different branch

**Comment by mekarpeles** [comment]
lgtm now

**Pull request merged**